feature/CourseView/Sources/CourseView/ExerciseTab/ExerciseParticipation/ModelingExerciseParticipation/ModelingExerciseView.swift

```diff
@@ -9,6 +9,7 @@ import ApollonEdit
 import DesignLibrary
 import SharedModels
 import SwiftUI
+import ApollonShared
 
 struct ModelingExerciseView: View {
```

Comment:
Sources should not be located in `feature/CourseView/Sources`, but in `ArtemisKit`

- ModelingExerciseView
- ModelingViewModel
- ExerciseSubmissionService
- ModelingExerciseSubmissionServiceImpl
- UnknownExerciseSubmissionServiceImpl